applications/app/controllers/AtomPageController.scala

```diff
@@ -79,8 +79,6 @@ class AtomPageController(contentApiClient: ContentApiClient, wsClient: WSClient,
     lookup(s"atom/$atomType/$id") map {
       case Left(model: MediaAtom) =>
         renderAtom(MediaAtomPage(model, withJavaScript = isJsEnabled, withVerticalScrollbar = hasVerticalScrollbar))
-      case Left(model: StoryQuestionsAtom) =>
-        renderAtom(StoryQuestionsAtomPage(model, withJavaScript = isJsEnabled, withVerticalScrollbar = hasVerticalScrollbar, inApp = inApp))
```

Comment:
Incidentally, this removes the last use of the `inApp: Boolean` parameter on the `render()` method, so this can now be removed. The parameter was added with https://github.com/guardian/frontend/pull/18213